Formula/bubblewrap.rb

```diff
@@ -0,0 +1,61 @@
+class Bubblewrap < Formula
+  desc "Unprivileged sandboxing tool for Linux"
+  homepage "https://github.com/containers/bubblewrap"
+  url "https://github.com/containers/bubblewrap/releases/download/v0.4.1/bubblewrap-0.4.1.tar.xz"
+  sha256 "b9c69b9b1c61a608f34325c8e1a495229bacf6e4a07cbb0c80cf7a814d7ccc03"
+  license "LGPL-2.0-or-later"
+
+  head do
+    url "https://github.com/containers/bubblewrap.git", branch: "master"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "docbook-xsl" => :build
+  depends_on "libxslt"     => :build
+  depends_on "libcap"
+  depends_on :linux
+
+  def install
+    system "autoreconf", "-fvi" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules",
+           "--with-bash-completion-dir=#{bash_completion}"
+
+    inreplace "Makefile" do |s|
+      s.gsub! "http://docbook.sourceforge.net/release/xsl/current",
+              "#{Formula["docbook-xsl"].opt_prefix}/docbook-xsl"
+    end
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/bwrap", "--version"
+    # Older Linux kernels don't always give namespace access to normal users:
+    userns = "1"
+    sysctl_unprivileged_userns_clone = "/proc/sys/kernel/unprivileged_userns_clone"
+    if File.exist?(sysctl_unprivileged_userns_clone)
+      userns = Utils.safe_popen_read("cat", sysctl_unprivileged_userns_clone).chomp
+    end
+    if userns == "1"
+      output = shell_output("unshare --map-root-user echo works ||" \
+                            "echo 'run: sudo sysctl kernel.unprivileged_userns_clone=1' >&2").chomp
+      userns = "not_enabled" if output != "works"
+    end
+    if userns == "1"
```

Comment:
All this checking needs to go, it should run succesfully on our CI and if people decide to run it locally the compatibility check is up to them.

Reply:
Unfortunatly not all github runners have the same kernel configuration:

Some enable unprivileged_userns_clone and some not.
This means, depending on whether the kernel the test process access to user-namespaces, a creation of a container can succeed or it cannot. Therefore it has to be checked what the kernel's configuration allows the test to do.

Is there a variable which shows the test if it runs on the github test runner or locally?

If people decide to run it locally and the kernel is configured to give access to user namespaces the test will run accordingly.

When the user tries to use bubblewrap and the kernel is later (this can change after the test was done), the bubblewrap command tells the user what he has to do to configure the system.

Reply:
I think you vastly overestimate the amount of people running brew test locally. It's basically just the maintainers and the author of the initial PR. You know when to run a command and the maintainers will almost exclusively run it in the brew docker.

Reply:
Removed the checks from the test as requested.